zkevm-circuits/src/witness/block.rs

```diff
@@ -68,6 +71,72 @@ impl<F: Field> Block<F> {
         }
     }
 }
+
+#[cfg(feature = "test")]
+use crate::exp_circuit::OFFSET_INCREMENT;
+#[cfg(feature = "test")]
+use crate::util::log2_ceil;
+
+#[cfg(feature = "test")]
+impl<F: Field> Block<F> {
```

Comment:
Doesn't this make the circuit configuration depend on the circuit witness input? Making it more likely that we end up with circuits that get a different setup for different blocks?

Like what we are talking here: https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/1025

Reply:
I think this is independent from #1025 as the params are mantained inside of the `Block` since we don't care about this issue here.

As for the test_degree, it's related only to the MockProver. The real prover uses fixed sizes. Hence, no chance that we end up with variadic-size circuits.